monai/handlers/mlflow_handler.py

```diff
@@ -222,6 +239,39 @@ def _set_experiment(self):
             raise ValueError(f"Cannot set a deleted experiment '{self.experiment_name}' as the active experiment")
         self.experiment = experiment
 
+    @staticmethod
```

Comment:
I don't get why make this function as static?

Thanks.

Reply:
Hi @Nic-Ma ,
    I thought this function didn't relate to any varible or method of this class and more like a utility function of MLFlow hander, so I made it static.

Thanks,
Bin